File: WebsocketServer/WebsocketServer/Utilities/BackendOps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WebSocketServer.Utilities
{
    static class BackendOps
    {
        #region ClientGroupBroadcastService
        /// <summary>
        /// 加入分组
        /// </summary>
        public const string Cmd_JoinGroup = "JoinGroup";

        /// <summary>
        /// 离开分组
        /// </summary>
        public const string Cmd_LeaveGroup = "LeaveGroup";

        /// <summary>
        /// 广播消息
        /// </summary>
        public const string Cmd_BroadcastMsg = "BroadcastMsg";
        #endregion

        #region CentralControllerService
        /// <summary>
        /// 注册成为监听者，监听服务事件
        /// </summary>
        public static string Cmd_RegisterAsListener = "RegisterAsListener";
        /// <summary>
        /// 从监听者注销
        /// </summary>
        public static string Cmd_UnregisterFromListener = "UnregisterFromListener";
        /// <summary>
        /// 注册一个 Master
        /// </summary>
        public static string Cmd_RegisterAsMaster = "RegisterAsMaster";
        /// <summary>
        /// 注销一个 Master
        /// </summary>
        public static string Cmd_UnregisterFromMaster = "UnregisterFromMaster";
        /// <summary>
        /// 当Master集合发生变化，增加，删减，修改
        /// </summary>
        public static string Notify_OnMasterCollectionChanged = "OnMasterCollectionChanged";
        /// <summary>
        /// 获取所有master
        /// </summary>
        public static string Cmd_GetAllMasters = "GetAllMasters";
        /// <summary>
        /// 注册一个slave
        /// </summary>
        public static string Cmd_RegisterAsSlave = "RegisterAsSlave";
        /// <summary>
        /// 注销一个slave
        /// </summary>
        public static string Cmd_UnregisterFromSlave = "UnregisterFromSlave";
        /// <summary>
        /// 发送消息
        /// </summary>
        public static string Cmd_Broadcast = "Broadcast";
        #endregion

        /// <summary>
        /// 网络心跳
        /// </summary>
        public const string WSPing = "WSPing";
    }
}

File: WebsocketServer/WebsocketServer/wwwroot/TypeScriptClient/lib/clientKernal/BackendOps.ts
namespace WebsocketTSClient
{
    export class BackendOps
    {
        /**
        * 加入分组
        */
        public static Cmd_JoinGroup: string = "JoinGroup";

        /**
        * 离开分组
        */
        public static Cmd_LeaveGroup: string = "LeaveGroup";

        /**
        * 广播消息
        */
        public static Cmd_BroadcastMsg: string = "BroadcastMsg";

        /**
          * 注册一个 Master
          */
        public static Cmd_RegisterAsListener: string = "RegisterAsListener";

        /**
        * 注销一个 Master
        */
        public static Cmd_UnregisterFromListener: string = "UnregisterFromListener";

        /**
        * 注册一个 Master
        */
        public static Cmd_RegisterAsMaster: string = "RegisterAsMaster";

        /**
        * 注销一个 Master
        */
        public static Cmd_UnregisterFromMaster: string = "UnregisterFromMaster";

        /**
        * 当Master集合发生变化，增加，删减，修改
        */
        public static Notify_OnMasterCollectionChanged: string = "OnMasterCollectionChanged";

        /**
        * 获取所有master
        */
        public static Cmd_GetAllMasters: string = "GetAllMasters";

        /**
        * 注册一个slave
        */
        public static Cmd_RegisterAsSlave: string = "RegisterAsSlave";

        /**
        * 注销一个slave
        */
        public static Cmd_UnregisterFromSlave: string = "UnregisterFromSlave";

        /**
        * 发送消息
        */
        public static Cmd_Broadcast: string = "Broadcast";

        /**
        * 网络心跳
        */
        public static WSPing: string = "WSPing";
    }
}